src/muya/lib/ui/imageToolbar/index.js

```diff
@@ -35,8 +37,9 @@ class ImageToolbar extends BaseFloat {
   listen () {
     const { eventCenter } = this.muya
     super.listen()
-    eventCenter.subscribe('muya-image-toolbar', ({ reference, imageInfo }) => {
+    eventCenter.subscribe('muya-image-toolbar', ({ reference, imageInfo, realFilePath }) => {
       this.reference = reference
+      this.realFilePath = realFilePath
```

Comment:
I don't know whether we should use `getImageInfo` instead the image source because the source may contain other tag information etc. Also, this would make it easier to understand that we have to remove `file://` prefix.

Reply:
getImageInfo can only take the relative path of the image, not the absolute path of the image.
Therefore, the src of taget is referenced here.